chainerx_cc/chainerx/context.h

```diff
@@ -61,6 +61,18 @@ class Context {
     // Gets the native backend.
     native::NativeBackend& GetNativeBackend();
 
+    // Registers the backend.
+    template <typename BackendType>
+    Backend& CreateBackend(const std::string& backend_name) {
+        auto backend = std::unique_ptr<Backend, context_detail::BackendDeleter>{
+                new BackendType{*this}, context_detail::BackendDeleter{[](gsl::owner<Backend*> ptr) { delete ptr; }}};
+        Backend& backend_ref = RegisterBackend(backend_name, std::move(backend));
+        if (&*backend != &backend_ref) {
```

Comment:
`&*backend` is illegal because it's moved out.
Use `unique_ptr::get` to get a pointer.